components/global/core/io.cellery.observability.api/src/main/java/io/cellery/observability/api/KubernetesAPI.java

```diff
@@ -59,6 +59,29 @@ public Response getK8sPods(@QueryParam("queryStartTime") long queryStartTime,
         }
     }
 
+    @GET
+    @Path("/components")
```

Comment:
The API should be changed as /cells/{cellName}/components/{componentName}

Reply:
Noted. I will update.